pkg/image/admission/admission.go

```diff
@@ -256,6 +256,7 @@ func (a *imageLimitRangerPlugin) getImageStreamImages(is, previousIS *imageapi.I
 // needle exists.
 func containsImage(needle imageapi.Image, haystack []imageapi.Image) bool {
 	for _, img := range haystack {
+		// TODO: miminar is this a sufficient comparison?
```

Comment:
Why not compare image names? (`img.Name`)
I'd prefer using `sets.String` with image names as values.

`img.DockerImageMetadata.ID` is becoming obsolete.